src/index.ts

```diff
@@ -261,6 +261,7 @@ commander
   .description('Shows the StackDriver logs')
   .option('--json', 'Show logs in JSON form')
   .option('--open', 'Open the StackDriver logs in browser')
+  .option('--setup', 'Setup StackDriver logs')
```

Comment:
This is OK, but eventually we'd this setup to be 1 step that enables logs, enabling APIs and executing functions.

As in, I don't know if we want:
```
clasp logs --setup
clasp apis --setup
clasp run --setup
```

in the future since you only need to setup once.

In reality, we're configuring this file:

https://github.com/google/clasp/blob/5cd29326fdf52a8577f5e284a61d38953275e0d4/src/utils.ts#L45-L49

We should probably have a command like
- `clasp settings --scriptId`
- `clasp settings --projectId`
- `clasp settings --rootDir`

---

For now, can we make the experience:
- Run clasp logs
  - If you don't have the project setup, prompt like you do in this PR.
  - If you have the project set up, run the command.